parquet-hadoop/src/main/java/org/apache/parquet/hadoop/MemoryManager.java

```diff
@@ -47,6 +48,15 @@
   private final long minMemoryAllocation;
   private final Map<InternalParquetRecordWriter, Long> writerList = new
       HashMap<InternalParquetRecordWriter, Long>();
+  private final Map<String, Runnable> callBacks = new HashMap<String, Runnable>();
+  private final MemoryManagerStats stats;
+
+  /**
+   * The state of MemoryManager. We could add more attributes in the future.
+   */
+  public static class MemoryManagerStats {
```

Comment:
Now that we aren't passing stats to the callback, what is the value of this class? Why not just add a `getScale` method instead of a `getStats` method?